src/main/java/liquibase/ext/databricks/change/createTable/ExtendedTableProperties.java

```diff
@@ -15,20 +23,4 @@ public String getSerializedObjectName() {
     public String getSerializedObjectNamespace() {
         return "http://www.liquibase.org/xml/ns/databricks";
```

Comment:
As this class is being refactored, here we could also return  `NamespaceDetailsDatabricks.DATABRICKS_NAMESPACE`